src/frame-writer.cpp

```diff
@@ -13,7 +13,7 @@
 
 
 static const AVRational US_RATIONAL{1,1000000} ;
-#define AUDIO_RATE 44100
+#define AUDIO_RATE 48000
```

Comment:
While we're at it, we should remove this #define since it's already defined in the header this file includes.